src/test/integration/fees.test.ts

```diff
@@ -144,20 +151,19 @@ describe('Ocean Node fees', () => {
       serviceId: service.id,
       command: PROTOCOL_COMMANDS.GET_FEES
     })
+    console.log('data log: ', data.status)
+    console.log('data stream: ', data.stream)
     expect(data.status.httpStatus).to.equal(200)
     const { stream } = data
     if (stream) {
       const buffer: any[] = []
-      stream.on('data', (data) => {
-        // read streamed data to buffer
-        buffer.push(data)
-      })
       stream.on('end', () => {
         // check that we got a valid response
         const feesData: ProviderFeeData = JSON.parse(buffer.toString()) as ProviderFeeData
+        console.log('fees data json: ', feesData)
         expect(feesData.providerFeeAddress).to.be.equal(address)
         expect(feesData.providerFeeToken).to.be.equal(providerFeeToken)
-        expect(feesData.providerFeeAmount).to.be.equal(providerAmount)
+        expect(feesData.providerFeeAmount).to.be.equal(1000000000000000000n)
         expect(feesData.v).to.be.gte(27) // 27 OR 28
```

Comment:
This line here could be improved to check that the value is actually 27 or 28

```suggestion
        assert(feesData.v === 27 || (feesData.v === 27, 'Expected feesData.v to be 27 or 28')
```

Reply:
this line is still wrong:
```
assert(
          feesData.v === 27 || (feesData.v === 27, 'Expected feesData.v to be 27 or 28')
        )
```
it checks 2 times for 27